templates/project/.gitattributes.twig

```diff
@@ -18,6 +18,10 @@ phpstan-baseline.neon export-ignore
 phpstan-console-application.php export-ignore
 psalm.xml export-ignore
 psalm-baseline.xml export-ignore
+{% if project.hasFrontend %}
+postcss.config.js export-ignore
+prettier.config.js export-ignore
+{% endif %}
```

Comment:
also webpack.config.js and package-lock.json

Reply:
I think we could treat them on a separate PR, for some reasons:
* First file that even if they are not handled via dev-kit, are added on gitatributes
* Why package-lock.json and no package.json?

If you don't mind, I merge this first to avoid removing files on SonataAdmin, and then we can talk about these.

Reply:
https://github.com/sonata-project/dev-kit/pull/1929